### ISSUE TITLE
[Feature Request]: redirect save and profile to alternate folders

### ISSUE THREAD

**Issue by k0pp0** [emoose/OutRun2006Tweaks#128]
### Describe the feature you'd like to see added

Would like the option to move the default save locations for the game (pretty please)

Idea: move save files to OneDrive/Dropbox to sync across devices. @emoose

**Comment by Tenome** [comment]
For now, I guess you can hardlink your saves folder to the Dropbox/Onedrive/Google Drive desktop volume so it syncs online. I used to do that ages ago.

**Comment by k0pp0** [comment]
Yeah, that was the backup option, but wondered if it could be picked instead.

**Comment by emoose** [comment]
Looks like there's just a single function that returns the savegame dir, would probably be pretty easy to redirect.

There's a few options we could use, any ideas which would be best?
- C:\Users\xxx\Documents\My Games\OutRun2006
- C:\Users\xxx\Saved Games\OutRun2006
- C:\Users\xxx\AppData\Local\OutRun2006

**Comment by Tenome** [comment]
AppData\Roaming is the best bet I think, that's where most modern games store the save file.

**Comment by k0pp0** [comment]
C:\Users\xxx\Documents\My Games\OutRun2006

This means the automatic one drive sync would pick it up. 

Or a value in the tweaks ini file?
